padertorch/ops/losses/regression.py

```diff
@@ -49,45 +49,59 @@ def log_mse_loss(estimate: torch.Tensor, target: torch.Tensor,
             TasNet: A Dissecting Approach.” ArXiv:1911.08895
             [Cs, Eess], November 20, 2019.
             http://arxiv.org/abs/1911.08895.
+
+    >>> estimate = [[1., 2, 3], [4, 5, 6]]
+    >>> target = [[2., 3, 4], [4, 0, 6]]
+    >>> log_mse_loss(torch.tensor(estimate), torch.tensor(target))
+    tensor(0.9208)
+    >>> log_mse_loss(torch.tensor(estimate), torch.tensor(target), reduction=None)
+    tensor([0.0000, 0.9208])
     """
     # Use the PyTorch implementation for MSE, should be the fastest
     return _reduce(
-        F.mse_loss(estimate, target, reduce='none').mean(dim=-1).log10(),
-        reduce=reduce
+        F.mse_loss(estimate, target, reduction='none').mean(dim=-1).log10(),
+        reduction=reduction
     )
 
 
-def sdr_loss(estimates: torch.Tensor, targets: torch.Tensor,
-             reduce: str = 'mean'):
+def sdr_loss(estimate: torch.Tensor, target: torch.Tensor,
+             reduction: str = 'mean'):
     """
     The (scale dependent) SDR or SNR loss.
 
     Args:
-        estimates (KxT):
-        targets (KxT, same as estimates):
+        estimate (... x T): The estimated signal
+        target (... x T, same as estimate): The target signal
+        reduction: 'mean', 'sum' or 'none'/None for batch dimensions
 
     Returns:
 
+    >>> estimate = [[1., 2, 3], [4, 5, 6]]
+    >>> target = [[2., 3, 4], [4, 0, 6]]
+    >>> sdr_loss(torch.tensor(estimate), torch.tensor(target))
+    tensor(-6.5167)
+    >>> sdr_loss(torch.tensor(estimate), torch.tensor(target), reduction=None)
+    tensor([-9.8528, -3.1806])
+
     """
     # Calculate the SNR. The square in the power computation is moved to the
     # front, thus the 20 in front of the log
     snr = 20 * torch.log10(
-        torch.norm(targets, dim=-1) / torch.norm(estimates - targets, dim=-1)
+        torch.norm(targets, dim=axis) / torch.norm(estimates - targets, dim=axis)
```

Comment:
Looks like `axis` is not defined here

Reply:
And `estimates` and `targets` are not defined

Reply:
Done